clusterloader2/pkg/measurement/common/network/manifests/worker-deployment.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  namespace: netperf
+  name: worker
+  labels:
+    group: network
+spec:
+  replicas: {{.Replicas}}
+  selector:
+    matchLabels:
+      name: worker
+  template:
+    metadata:
+      labels:
+        group: network
+        name: worker
+    spec:
+      containers:
+      - image:  vivekthrivikraman/netperfbenchmark:0.1
+        name: worker
+        imagePullPolicy: IfNotPresent
+        resources:
+          limits:
+            cpu: "1"
```

Comment:
let's set limits and requests to the same value - that will make it more predictible

Reply:
ok done.